modules/uptime-check/main.tf

```diff
@@ -5,49 +5,40 @@ locals {
 
 resource "google_monitoring_uptime_check_config" "uptime_check_config" {
   for_each = { for i in var.uptime_checks : i.service_name => i }
-
   display_name = each.value.service_name
+  project      = var.monitoring_project_id
   timeout      = lookup(each.value, "timeout", local.default_timeout)
   period       = lookup(each.value, "period", local.default_period)
 
   selected_regions = lookup(each.value, "regions", []) //Global used by default
 
-  project = var.monitoring_project_id
-
-  dynamic "http_check" {
-    for_each = (lookup(each.value, "check_type", null) != null) ? [each.value.check_type] : []
-    content {
-      path           = lookup(each.value, "path", "/")
-      port           = (lookup(each.value, "check_type", null) == "HTTP" ? "80" : "443")
-      use_ssl        = (lookup(each.value, "check_type", null) == "HTTP" ? false : true)
-      validate_ssl   = (lookup(each.value, "check_type", null) == "HTTP" ? false : true)
-      request_method = lookup(each.value, "request_method", "GET")
-      dynamic "auth_info" {
-        for_each = (lookup(each.value, "password", null) != null) ? [each.value.password] : []
-        content {
-          password = lookup(each.value, "password", null)
-          username = lookup(each.value, "username", null)
-        }
+  http_check {
+    path           = lookup(each.value, "path", "/health")
+    port           = (lookup(each.value, "check_type", null) == "HTTP" ? "80" : "443")
+    use_ssl        = (lookup(each.value, "check_type", null) == "HTTP" ? false : true)
+    validate_ssl   = (lookup(each.value, "check_type", null) == "HTTP" ? false : true)
+    request_method = lookup(each.value, "request_method", "GET")
+    dynamic "auth_info" {
+      for_each = lookup(each.value, "password", [])
+      content {
+        password = lookup(each.value, "password", null)
+        username = lookup(each.value, "username", null)
       }
-      headers      = (lookup(each.value, "headers", null) != null) ? each.value.headers : null
-      body         = lookup(each.value, "body", null)
-      content_type = lookup(each.value, "content_type", null) 
     }
+    headers      = lookup(each.value, "headers", {})
+    body         = lookup(each.value, "body", null)
+    content_type = lookup(each.value, "content_type", null)
   }
 
-  dynamic "monitored_resource" {
-    for_each = (lookup(each.value, "type", null) != null) ? [each.value.type] : []
-    content {
-      type   = lookup(each.value, "type", "uptime_url")
-      labels = {
-        host       = lookup(each.value, "hostname", null)
-        project_id = var.project_id
-      }
+  monitored_resource {
+    type = lookup(each.value, "type", "uptime_url")
+    labels = {
+      host       = lookup(each.value, "hostname", null)
```

Comment:
terraform fmt

Reply:
i think we have terraform fmt in all examples or should we change to reading a file?

Reply:
`host       = lookup(each.value, "hostname", null)` -> `host = lookup(each.value, "hostname", null)`